shpc/tests/test_wrappers.py

```diff
@@ -0,0 +1,118 @@
+#!/usr/bin/python
+
+# Copyright (C) 2022 Vanessa Sochat.
+
+# This Source Code Form is subject to the terms of the
+# Mozilla Public License, v. 2.0. If a copy of the MPL was not distributed
+# with this file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import pytest
+import os
+
+import shpc.main.container as container
+import shpc.main.wrappers.base as wrappers_base
+
+from .helpers import init_client, here
+
+
+@pytest.mark.parametrize(
+    "with_custom_wrapper_template_path,include_container_dir",
+    [
+        (False, False),
+        (False, True),
+        (True, False),
+        (True, True),
+    ],
+)
+def test_get_template_paths(
+    tmp_path, with_custom_wrapper_template_path, include_container_dir
+):
+    """
+    Test the shpc.main.wrappers.base.WrapperScript.get_template_paths function
+    """
+    # Note: the module_sys and container_tech parameters are actually not used in the test
+    client = init_client(str(tmp_path), "tcl", "singularity")
+    # The file name is actually not used in the test, only its enclosing directory
+    testdata_dir = os.path.join(here, "testdata")
+    container_yaml = os.path.join(testdata_dir, "quay-container.yaml")
+    config = container.ContainerConfig(container_yaml)
+    ws = wrappers_base.WrapperScript(
+        "dummy.sh",  # Not used in this test
+        client.settings,
+        None,  # image
+        None,  # container
+        config,
+    )
+    client.settings.set(
+        "wrapper_scripts",
+        "templates:" + (str(tmp_path) if with_custom_wrapper_template_path else "None"),
+    )
+
+    # get_template_paths
+    # - always default template path
+    # - optional wrapper path from settings
+    # - optional cf include_container_dir
+    tp = ws.get_template_paths(include_container_dir)
+    expected_len = (
+        1 + int(with_custom_wrapper_template_path) + int(include_container_dir)
+    )
+    assert len(tp) == expected_len
+    if with_custom_wrapper_template_path:
+        assert str(tmp_path) in tp
+    else:
+        assert str(tmp_path) not in tp
+    if include_container_dir:
+        assert testdata_dir in tp
+    else:
+        assert testdata_dir not in tp
+
+
+@pytest.mark.parametrize(
+    "make_absolute,exists",
+    [
+        (False, False),
+        (False, True),
+        (True, False),
+        (True, True),
+    ],
+)
+def test_find_wrapper_script(tmp_path, make_absolute, exists):
+    """
+    Test views:
```

Comment:
Should this comment be test (something else?)

Reply:
Yep. Forgot to change it